src/app/Layout/Layout.tsx

```diff
@@ -0,0 +1,74 @@
+"use client"
+import React, {  useState } from 'react'
+import {
+    MagnifyingGlassIcon,
+    HamburgerMenuIcon,
+} from '@radix-ui/react-icons'
+import { Input } from "@/components/ui/input"
+import { Button } from "@/components/ui/button"
+import { Sidebar } from '../components/Sidebar'
+import { usePathname } from 'next/navigation'
+import { RoutesTitle } from '@/lib/routes'
+
+type UserRole = "Incharge" | "member" | "Shift-Incharge";
```

Comment:
Bilal, I'm not sure if these are the right values. Also, they shouldn't be in `Layout` component because they'll be reused across different places. Help Nadir here. Ping me if you're not sure how to fix this.

Reply:
This issue was resolved by creating a common type file. Now, whenever we use these values in our files, we can easily import the types from that shared file.